Traceback (most recent call last):
  File "/opt/homebrew/Caskroom/miniconda/base/envs/surv/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3621, in get_loc
    return self._engine.get_loc(casted_key)
  File "pandas/_libs/index.pyx", line 136, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 142, in pandas._libs.index.IndexEngine.get_loc
TypeError: '(array([  4,   5,   6,   8,   9,  11,  12,  18,  20,  21,  22,  23,  27,
        29,  30,  31,  32,  36,  38,  39,  40,  42,  44,  47,  51,  56,
        57,  58,  59,  61,  62,  63,  65,  66,  67,  68,  69,  74,  76,
        78,  79,  81,  83,  84,  85,  89,  90,  96,  97, 102, 104, 106,
       112, 115, 117, 118, 119, 122, 124, 125, 126, 129, 130, 131, 132,
       133, 134, 135, 136, 145, 146, 148, 152, 153, 154, 156, 157, 159,
       160, 162, 165, 167, 168, 169, 170, 172, 173, 178, 179, 182, 183,
       184, 185, 186, 187, 190, 194, 195, 197]), slice(None, None, None))' is an invalid key
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/jahan/Documents/cardiacdeepsurv/train.py", line 47, in <module>
    model.fit(X[train_index, :], y[train_index])
  File "/opt/homebrew/Caskroom/miniconda/base/envs/surv/lib/python3.10/site-packages/pandas/core/frame.py", line 3505, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/opt/homebrew/Caskroom/miniconda/base/envs/surv/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3628, in get_loc
    self._check_indexing_error(key)
  File "/opt/homebrew/Caskroom/miniconda/base/envs/surv/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 5637, in _check_indexing_error
    raise InvalidIndexError(key)
pandas.errors.InvalidIndexError: (array([  4,   5,   6,   8,   9,  11,  12,  18,  20,  21,  22,  23,  27,
        29,  30,  31,  32,  36,  38,  39,  40,  42,  44,  47,  51,  56,
        57,  58,  59,  61,  62,  63,  65,  66,  67,  68,  69,  74,  76,
        78,  79,  81,  83,  84,  85,  89,  90,  96,  97, 102, 104, 106,
       112, 115, 117, 118, 119, 122, 124, 125, 126, 129, 130, 131, 132,
       133, 134, 135, 136, 145, 146, 148, 152, 153, 154, 156, 157, 159,
       160, 162, 165, 167, 168, 169, 170, 172, 173, 178, 179, 182, 183,
       184, 185, 186, 187, 190, 194, 195, 197]), slice(None, None, None))